yrs/src/block.rs

```diff
@@ -51,7 +60,7 @@ impl BlockPtr {
     }
 }
 
-#[derive(Debug, PartialEq)]
+#[derive(Debug, Clone, PartialEq)]
```

Comment:
Atm. integration process does deep cloning. It's because Rust's `Vec`/`VecDeque` only allow to take elements from back/front. I've got an idea that's basically something like non-resizable vec, that tracks the occupation of the indexes, this way allowing us to remove elements from the middle of it. I'll add it in the separate PR.